cmd/neofs-ir/defaults.go

```diff
@@ -38,10 +38,8 @@ func defaultConfiguration(cfg *viper.Viper) {
 	cfg.SetDefault("logger.level", "info")
 
 	cfg.SetDefault("profiler.address", "localhost:6060")
-	cfg.SetDefault("profiler.shutdown_timeout", "30s")
 
 	cfg.SetDefault("metrics.address", "localhost:9090")
```

Comment:
This used the old values because of the need to support both sections.
I think we can remove already deprecated sections in the next release and simplify the code considerably.